js/extensions/footer.js

```diff
@@ -1,20 +1,18 @@
 export default function footer() {
-  /* --footer------------------- */
-
-  let footer = document.createElement("footer");
-  let footerContainer = document.createElement("div");
+  const footer = document.createElement("footer");
+  const footerContainer = document.createElement("div");
   footerContainer.className = "container";
 
-  let footerContent = document.createElement("p");
-  let date = new Date();
+  const footerContent = document.createElement("p");
+  const date = new Date();
   footerContent.className = "flex j-c-c align-items-c gap-05-rem";
   footerContent.innerHTML = `&copy;${date.getFullYear()} Deployed By`;
```

Comment:
Why do you create elements using JS instead of creating them directly in the index.html file?

Reply:
I did not updated this application for a while, thus you for the notes and contribution